### PR TITLE
Task/frontend258 comment crud

### PR DESCRIPTION
**Pull request by Barracuda713**

### CODE REVIEW

**Comment by Barracuda713** [comment]
[Implement comment-replies update/delete feature](https://trello.com/c/leMqbxyw/258-implement-comment-replies-update-delete-feature)

File: backend/core/src/primary-adapters/rest/comment/comment-controller.ts [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    if (comment.authorId !== user.id) {
+      throw new Forbidden(exceptionMessages.comment.COMMENT_FORBIDDEN);
+    }
+
+    if (comment.repliesCount > 0) {
+      const videoComment = await this.commentService.updateCommentById(id, '', true);
+      if (!videoComment) {
+        throw new NotFound(exceptionMessages.comment.COMMENT_NOT_FOUND);
+      }
+      return trimComment(videoComment);
```
Comment (WanderningMaster) [inline backend/core/src/primary-adapters/rest/comment/comment-controller.ts]: I guess it would be better create DTO on service layer

**Review by foxt451 — DISMISSED** [review]

File: frontend/src/pages/video/common/comment-block/config.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const matchMenuOptionWithIconName: Record<CommentMenuOptions, IconName> = {
+  [CommentMenuOptions.EDIT]: IconName.EDIT,
+  [CommentMenuOptions.DELETE]: IconName.DELETE,
+  // [CommentMenuOptions.REPORT]: IconName.REPORT,
+};
+
+const matchMenuOptionWithText: Record<CommentMenuOptions, string> = {
+  [CommentMenuOptions.EDIT]: 'Edit',
+  [CommentMenuOptions.DELETE]: 'Delete',
+  // [CommentMenuOptions.REPORT]: 'Report',
```
Comment (WanderningMaster) [inline frontend/src/pages/video/common/comment-block/config.ts:12]: is it should be used in future or you just forget remove this?

**Review by WanderningMaster — CHANGES_REQUESTED** [review]

File: frontend/src/pages/video/common/comment-block/config.ts (hunk 2 above)
Comment (Barracuda713) [reply frontend/src/pages/video/common/comment-block/config.ts:12]: @WanderningMaster it will be used in future.
I commented it, only for now.

**Review by WanderningMaster — APPROVED** [review]

**Pull request merged**